doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,38 @@
 # 4. CNF Test Cases and Requirements Traceability
 
 ## Table of Contents
+* [4.1 Introduction](#4.1)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1 Test Case Traceability to RA2 Requirements](#4.3.1)
+  
+<a name="4.1"></a>
+## 4.1 Introduction
+The scope of this chapter is to identify and list down test cases based on requirements defined in [Reference Architecture-2 (RA-2)](../../../ref_arch/kubernetes/README.md). This will serve as traceability between test cases and requirements for Kubernetes platform interoperability.
+
+Note that each requirement may have one or more test cases associated with it.
+
+<a name="4.2"></a>
+## 4.2 Selection Criteria
+Test cases and their dependencies must be open source. The test cases (or test suite with the test case) as well as the environment needed to run the test should be reproducible by any party following publicly available documentation.
```

Comment:
"Test cases and their dependencies must be open source" -- is it test cases to be open source or Test Tools?